test/test_dataaccess.py

```diff
@@ -160,6 +195,21 @@ def test_open_data(self):
         self.assertEqual({'latitude', 'longitude', 'time'}, set(dataset.AOD550.dims))
         self.assertEqual({20, 20, 1}, set(dataset.AOD550.chunk_sizes))
 
+    @skipIf(os.environ.get(
+        'XCUBE_DISABLE_WEB_TESTS', None) == '1', 'XCUBE_DISABLE_WEB_TESTS = 1')
+    def test_open_data_with_crs(self):
+        dataset = self.opener.open_data(
+            'esacci.ICESHEETS.yr.Unspecified.IV.PALSAR.ALOS.UNSPECIFIED.1-1.'
+            'greenland_margin_2006_2011',
+        )
+        self.assertIsNotNone(dataset)
+        self.assertTrue('crs' in dataset.data_vars.keys())
+        self.assertTrue('grid_mapping_name' in dataset['crs'].attrs)
+
+        grid_mapping = GridMapping.from_dataset(dataset)
+        self.assertIsNotNone(grid_mapping)
```

Comment:
`GridMapping.from_dataset(dataset)` will never return `None`. It raises a `ValueError` instead.
It does make more sense to verify that the returned grid mapping's properties are the ones you expect .

Reply:
Done